README.md

```diff
@@ -251,6 +251,53 @@ Join our [Google Group](https://groups.google.com/forum/#!forum/medic-developers
 
 You can contribute by messaging our [Google Group](https://groups.google.com/forum/#!forum/medic-developers), raising bugs or improvements in the medic-webapp Github repo, or submitting pull requests.
 
+## Vulnerability Disclosure Policy
+
+We take the security of our systems seriously, and we value the security community. The disclosure of security vulnerabilities helps us ensure the security and privacy of our users.
+
+### Guidelines
+
+We require that all researchers:
+
+- Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction of data during security testing;
```

Comment:
I _might_ add "Refrain from using any in-scope compromise as a platform to probe or conduct additional research, on any other system, regardless of scope", or something to that effect. Basically a "don't launch attacks from a container if you pwn it".